src/common/Button/Button.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import { ButtonProps } from "../interfaces";
+
+const Button = (props: ButtonProps) => {
+  const { onClick, clsx, children } = props;
+
+  return (
+    <button onClick={onClick} className={clsx ? clsx : ""}>
```

Comment:
By `clsx` I meant the npm package for handling classes https://www.npmjs.com/package/clsx

Reply:
```ts
import clsx from 'clsx';

const Button = (props: ButtonProps) => {
    const { className } = props;
    <button className={clsx('some-local-class', className)}>Hello</button>
}
```